divvy/cloudfilebrowser.js

```diff
@@ -720,6 +734,56 @@ var cloudFileBrowser = (function() {
             }
         },
 
+        performSearch: function(keywoard, element, path) {
+            var callbackArgs = {
+                'element' : element,
+                'path' : '/'
+            };
+
+            cloudFileBrowser.showLoading();
+
+            if(keywoard === '' || keywoard === undefined) {
```

Comment:
keyword*

Reply:
Also, please add some comments explaining the difference between no keyword and keyword. Also, what's going to be the most common case? We should add that case first.